plugins/inputs/influxdb_listener/influxdb_listener_test.go

```diff
@@ -0,0 +1,115 @@
+package http_listener
+
+import (
+	"fmt"
+	"net/http"
+	"net/http/httptest"
+	"strings"
+	"testing"
+	"time"
+
+	"github.com/influxdata/telegraf/internal"
+	"github.com/influxdata/telegraf/plugins/parsers/influx"
+	"github.com/influxdata/telegraf/selfstat"
+	"github.com/influxdata/telegraf/testutil"
+)
+
+// newListener is the minimal HTTPListener construction to serve writes.
+func newListener() *HTTPListener {
+	listener := &HTTPListener{
+		TimeFunc:  time.Now,
+		acc:       &testutil.NopAccumulator{},
+		BytesRecv: selfstat.Register("http_listener", "bytes_received", map[string]string{}),
+		handler:   influx.NewMetricHandler(),
+		pool:      NewPool(200, DEFAULT_MAX_LINE_SIZE),
+		MaxLineSize: internal.Size{
+			Size: DEFAULT_MAX_LINE_SIZE,
+		},
+		MaxBodySize: internal.Size{
+			Size: DEFAULT_MAX_BODY_SIZE,
+		},
+	}
+	listener.parser = influx.NewParser(listener.handler)
+	return listener
+}
+
+func BenchmarkHTTPListener_serveWrite(b *testing.B) {
```

Comment:
Is the underscore normal here?  It looks kind of out of place.

Reply:
Hey @reimda I was following the format of Examples like:

```
 The naming convention to declare examples for the package, a function F, a type T and method M on type T are:

func Example() { ... }
func ExampleF() { ... }
func ExampleT() { ... }
func ExampleT_M() { ... }
```